src/server/templates/page.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Render a full HTML page
+ * @param {Object} params
+ * @param {?string} params.title Title of the page
+ * @param {string} [params.body = ""] HTML to render in the body of the page
+ * @param {Object.<string, Object.<string, string>>} params.assets Manifest of filename entry points to bundled assets.
+ * @return {string} HTML
```

Comment:
Oh damn I didn't know about that, I wrongly assumed that no marker was the same as the ! marker. I've added it to my TODO list to fix this jsdoc